client/src/components/Cities.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {Link} from 'react-router-dom';
+import {getCities} from '../actions/citiesActions';
+import {connect} from 'react-redux'; 
+
+class Cities extends Component {
+    constructor() {
+      super();
+      this.state = {
+        search: '',
+        //cities:[]
+      };
+    }
+
+    updateSearch(event) {
+      this.setState({search: event.target.value});
+    }
+
+  componentDidMount() {
+    // axios.get("/api/city").then(res => {
```

Comment:
Generally its best to remove commented code unless you plan on reintroducing it between commits.  Only because it may lead to confusion later on.